extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -61,6 +61,14 @@
     @ConfigItem
     public Optional<Duration> accessControlMaxAge;
 
+    /**
+     * The `Access-Control-Allow-Credentials` header is used to tell the
+     * browsers to expose the response to front-end JavaScript code when
+     * the request’s credentials mode Request.credentials is “include”.
+     */
+    @ConfigItem(defaultValueDocumentation = "Default value is true when origins is present and does not include '*', else it is false")
+    public Optional<Boolean> accessControlAllowCredentials;
```

Comment:
I think it should be `boolean`, with the `defaultValue=false`

Reply:
My rational here was that the user may want to force `false`. It can be strange to define the value as false, and in fact becoming true because the conditions match. No?

Reply:
@mcserra, makes sense

Reply:
Ok, cool :) So you want me to revert it?